components/feature/downloads/src/main/res/values/strings.xml

```diff
@@ -11,4 +11,7 @@
 
     <!-- Alert dialog confirmation before download a file, this is the negative action. -->
     <string name="mozac_feature_downloads_dialog_cancel">Cancel</string>
+
+    <!-- Error shown when the user is trying to download a invalid file. -->
+    <string name="mozac_feature_downloads_file_not_supported">Download protocol not supported</string>
```

Comment:
Is that the message that Focus uses? Otherwise I'm wondering if we should include the protocol in the error message.

Reply:
It really came from [FireFoxLite](https://github.com/mozilla-tw/FirefoxLite/blob/fb7a5e18043d88378be3db7e350470a125513cb3/app/src/main/java/org/mozilla/focus/download/EnqueueDownloadTask.java#L139), [focus is just ignoring the download and not showing an error message](https://github.com/mozilla-mobile/focus-android/blob/0bc2d3dc8db69364fa4715f6382e5b039984bb5b/app/src/main/java/org/mozilla/focus/web/GeckoWebViewProvider.kt#L387). Should I update the message? If that it's the case which format should I use for the message?

Reply:
Okay, let's keep the string you have here for now! :)

Reply:
👍